db/migrate/20181127010456_add_https_to_icons.rb

```diff
@@ -0,0 +1,16 @@
+class AddHttpsToIcons < ActiveRecord::Migration[5.2]
+  def up
+    Icon.where("url LIKE ''").where("url LIKE ''").find_each do |icon|
```

Comment:
Uh, did you forget to fill this line in?

Reply:
...I am mystified because I thought I copied the _second_ bit from the _first_ bit after copying all of it from my console and changing URLs but WHATEVER I'M FIXING IT